### PR TITLE
Fix oh-my-zsh (fix #567)

### LINKED ISSUE #567: Latest oh-my-zsh update, broke updating from script.
**What did you expect to happen?**
OhMy Zsh update normally

**What actually happened?**
this happened

**Additional details:**
- Fedora 33
- Cargo install

<details><pre>
topgrade -v
 2020-11-30T19:13:14.980Z DEBUG topgrade::config > Configuration at /home/mark/.config/topgrade.toml
 2020-11-30T19:13:14.980Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: None, ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2020-11-30T19:13:14.980Z DEBUG topgrade::utils  > Detected "/usr/bin/notify-send" as "notify-send"
 2020-11-30T19:13:14.981Z DEBUG topgrade         > Version: 6.0.0
 2020-11-30T19:13:14.981Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2020-11-30T19:13:14.981Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2020-11-30T19:13:14.981Z DEBUG topgrade         > Binary path: Ok("/home/mark/.cargo/bin/topgrade")
 2020-11-30T19:13:14.981Z DEBUG topgrade         > Self Update: false
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Cannot find "powershell"
 2020-11-30T19:13:14.981Z DEBUG topgrade::runner > Step "System update"

―― 22:13:14 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Path "/usr/bin/rpm-ostree" doesn't exist
 2020-11-30T19:13:14.981Z DEBUG topgrade::utils  > Path "/usr/bin/dnf" exists
 2020-11-30T19:13:14.981Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/dnf" "upgrade"
[sudo] password for mark: 
Last metadata expiration check: 0:22:00 ago on Mon 30 Nov 2020 09:51:28 PM MSK.
Dependencies resolved.
Nothing to do.
Complete!
 2020-11-30T19:13:29.591Z DEBUG topgrade::runner   > Step "etc-update"
 2020-11-30T19:13:29.591Z DEBUG topgrade::runner   > Step "Brew"
 2020-11-30T19:13:29.591Z DEBUG topgrade::utils    > Detected "/home/linuxbrew/.linuxbrew/bin/brew" as "brew"

―― 22:13:29 - Brew ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:29.992Z TRACE topgrade::executor > Output of "/home/linuxbrew/.linuxbrew/bin/brew" "--repository" "buo/cask-upgrade": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/buo/homebrew-cask-upgrade\n", stderr: "" }
 2020-11-30T19:13:29.992Z DEBUG topgrade::executor > Running "/home/linuxbrew/.linuxbrew/bin/brew" "update"
Already up-to-date.
 2020-11-30T19:13:32.129Z DEBUG topgrade::executor > Running "/home/linuxbrew/.linuxbrew/bin/brew" "upgrade" "--ignore-pinned"
 2020-11-30T19:13:34.362Z DEBUG topgrade::runner   > Step "yadm"
 2020-11-30T19:13:34.362Z DEBUG topgrade::runner   > Step "nix"
 2020-11-30T19:13:34.363Z DEBUG topgrade::runner   > Step "home-manager"
 2020-11-30T19:13:34.363Z DEBUG topgrade::runner   > Step "asdf"
 2020-11-30T19:13:34.363Z DEBUG topgrade::utils    > Path "/home/mark/.config/emacs" doesn't exist
 2020-11-30T19:13:34.363Z DEBUG topgrade::utils    > Path "/home/mark/.emacs.d" doesn't exist
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > /home/mark/.doom.d does not exists
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > /home/mark/.vim does not exists
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > /home/mark/.config/nvim does not exists
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > /home/mark/.zshrc is a file. Checking /home/mark
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > Checking if /home/mark is a git repository
 2020-11-30T19:13:34.363Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n" }
 2020-11-30T19:13:34.363Z DEBUG topgrade::steps::git > /home/mark/.tmux does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::steps::git > /home/mark/.config/fish does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::steps::git > /home/mark/.config/openbox does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::steps::git > /home/mark/.config/bspwm does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::steps::git > /home/mark/.config/i3 does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::steps::git > /home/mark/.config/sway does not exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "Git repositories"
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "zr"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "antibody"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "antigen"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "zplug"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "zinit"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Path "/home/mark/.zshrc" exists
 2020-11-30T19:13:34.364Z DEBUG topgrade::runner     > Step "zim"
 2020-11-30T19:13:34.364Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.365Z TRACE topgrade::executor   > Output of "zsh" "-c" "[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2020-11-30T19:13:34.365Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2020-11-30T19:13:34.365Z DEBUG topgrade::utils      > Detected "/usr/bin/zsh" as "zsh"
 2020-11-30T19:13:34.365Z DEBUG topgrade::utils      > Path "/home/mark/.oh-my-zsh" exists

―― 22:13:34 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:34.366Z TRACE topgrade::executor   > Output of "zsh" "-c" "test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2020-11-30T19:13:34.366Z DEBUG topgrade::steps::zsh > Running zsh returned exit code: 1: . Using default path: /home/mark/.oh-my-zsh/custom
 2020-11-30T19:13:34.366Z DEBUG topgrade::steps::zsh > oh-my-zsh custom dir: /home/mark/.oh-my-zsh/custom
 2020-11-30T19:13:34.366Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom is a git repository
 2020-11-30T19:13:34.367Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.367Z DEBUG topgrade::steps::git > /home/mark/.oh-my-zsh/custom/example.zsh is a file. Checking /home/mark/.oh-my-zsh/custom
 2020-11-30T19:13:34.367Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom is a git repository
 2020-11-30T19:13:34.367Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.367Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins is a git repository
 2020-11-30T19:13:34.369Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.369Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/example is a git repository
 2020-11-30T19:13:34.370Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.370Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/autoupdate is a git repository
 2020-11-30T19:13:34.371Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/autoupdate\n", stderr: "" }
 2020-11-30T19:13:34.371Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-history-substring-search is a git repository
 2020-11-30T19:13:34.372Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-history-substring-search\n", stderr: "" }
 2020-11-30T19:13:34.372Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting is a git repository
 2020-11-30T19:13:34.373Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting\n", stderr: "" }
 2020-11-30T19:13:34.373Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-completions is a git repository
 2020-11-30T19:13:34.373Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-completions\n", stderr: "" }
 2020-11-30T19:13:34.373Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions is a git repository
 2020-11-30T19:13:34.375Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions\n", stderr: "" }
 2020-11-30T19:13:34.375Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes is a git repository
 2020-11-30T19:13:34.375Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.375Z DEBUG topgrade::steps::git > /home/mark/.oh-my-zsh/custom/themes/example.zsh-theme is a file. Checking /home/mark/.oh-my-zsh/custom/themes
 2020-11-30T19:13:34.375Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes is a git repository
 2020-11-30T19:13:34.377Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh\n", stderr: "" }
 2020-11-30T19:13:34.377Z DEBUG topgrade::steps::git > Checking if /home/mark/.oh-my-zsh/custom/themes/powerlevel10k is a git repository
 2020-11-30T19:13:34.377Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/mark/.oh-my-zsh/custom/themes/powerlevel10k\n", stderr: "" }
Pulling custom plugins and themes
 2020-11-30T19:13:34.378Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.380Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.380Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.381Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.383Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.383Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-11-30T19:13:34.384Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "2c3bcd8b5d301f469fd8095d32ab2dc18e254a8e\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/themes/powerlevel10k
 2020-11-30T19:13:34.386Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "5ba8ae00cab1d59cccd828557b1f92b973e1d3c0\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/autoupdate
 2020-11-30T19:13:34.387Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "2e009c7ab8e9c7496ed86473ede0917a3cefee01\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-completions
 2020-11-30T19:13:34.388Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "0f80b8eb3368b46e5e573c1d91ae69eb095db3fb\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-history-substring-search
 2020-11-30T19:13:34.389Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "1715f39a4680a27abd57fc30c98a95fdf191be45\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
 2020-11-30T19:13:34.391Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ae315ded4dba10685dbbafbfa2ff3c1aefeb490d\n", stderr: "" }
Pulling /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2020-11-30T19:13:34.894Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "5ba8ae00cab1d59cccd828557b1f92b973e1d3c0\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/autoupdate
 2020-11-30T19:13:34.930Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "0f80b8eb3368b46e5e573c1d91ae69eb095db3fb\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-history-substring-search
 2020-11-30T19:13:34.936Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "2c3bcd8b5d301f469fd8095d32ab2dc18e254a8e\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/themes/powerlevel10k
 2020-11-30T19:13:34.996Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ae315ded4dba10685dbbafbfa2ff3c1aefeb490d\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2020-11-30T19:13:35.055Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "2e009c7ab8e9c7496ed86473ede0917a3cefee01\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-completions
 2020-11-30T19:13:35.082Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "1715f39a4680a27abd57fc30c98a95fdf191be45\n", stderr: "" }
Up-to-date /home/mark/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
 2020-11-30T19:13:35.082Z DEBUG topgrade::executor   > Running "sh" "/home/mark/.oh-my-zsh/tools/upgrade.sh"
/home/mark/.oh-my-zsh/tools/upgrade.sh: line 8: local: can only be used in a function
/home/mark/.oh-my-zsh/tools/upgrade.sh: line 9: local: can only be used in a function
/home/mark/.oh-my-zsh/tools/upgrade.sh: line 34: syntax error near unexpected token `('
/home/mark/.oh-my-zsh/tools/upgrade.sh: line 34: `  https://github.com/robbyrussell/oh-my-zsh(|.git))'
 2020-11-30T19:13:35.088Z DEBUG topgrade::terminal   > Desktop notification: oh-my-zsh failed

Retry? (y)es/(N)o/(s)hell
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "fisher"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "tmux"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "TLDR"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "pearl"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2020-11-30T19:13:39.653Z DEBUG topgrade::utils      > Detected "/usr/bin/bash" as "bash"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "apm"
 2020-11-30T19:13:39.653Z DEBUG topgrade::runner     > Step "rustup"
 2020-11-30T19:13:39.653Z DEBUG topgrade::utils      > Detected "/home/mark/.cargo/bin/rustup" as "rustup"

―― 22:13:39 - rustup ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:39.653Z DEBUG topgrade::executor   > Running "/home/mark/.cargo/bin/rustup" "self" "update"
info: checking for self-updates
  rustup unchanged - 1.23.0

 2020-11-30T19:13:39.883Z DEBUG topgrade::executor   > Running "/home/mark/.cargo/bin/rustup" "update"
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: checking for self-updates

  stable-x86_64-unknown-linux-gnu unchanged - rustc 1.48.0 (7eac88abb 2020-11-16)

info: cleaning up downloads & tmp directories
 2020-11-30T19:13:40.138Z DEBUG topgrade::runner     > Step "choosenim"
 2020-11-30T19:13:40.138Z DEBUG topgrade::runner     > Step "cargo"
 2020-11-30T19:13:40.138Z DEBUG topgrade::utils      > Detected "/home/mark/.cargo/bin/cargo-install-update" as "cargo-install-update"

―― 22:13:40 - Cargo ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:40.138Z DEBUG topgrade::executor   > Running "/home/mark/.cargo/bin/cargo-install-update" "install-update" "--git" "--all"
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package       Installed  Latest  Needs update
cargo-cache   v0.5.1     v0.5.1  No
cargo-expand  v1.0.4     v1.0.4  No
cargo-update  v4.1.2     v4.1.2  No
topgrade      v6.0.0     v6.0.0  No

No packages need updating.
Package  Installed  Latest  Needs update

No git packages need updating.
Overall updated 0 packages.
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "Flutter"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "Go"
 2020-11-30T19:13:44.518Z DEBUG topgrade::utils      > Detected "/usr/bin/go" as "go"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "Emacs"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "opam"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "vcpkg"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "pipx"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "stack"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "tlmgr"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "myrepos"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "jetpack"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "vim"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "Neovim"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "voom"
 2020-11-30T19:13:44.518Z DEBUG topgrade::runner     > Step "npm"
 2020-11-30T19:13:44.518Z DEBUG topgrade::utils      > Detected "/usr/bin/npm" as "npm"
 2020-11-30T19:13:44.741Z TRACE topgrade::executor   > Output of "/usr/bin/npm" "root" "-g": Output { status: ExitStatus(ExitStatus(0)), stdout: "/usr/local/lib/node_modules\n", stderr: "" }
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "yarn"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "deno"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "composer"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "krew"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "gem"
 2020-11-30T19:13:44.741Z DEBUG topgrade::utils      > Detected "/usr/bin/gem" as "gem"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "sheldon"
 2020-11-30T19:13:44.741Z DEBUG topgrade::runner     > Step "rtcl"
 2020-11-30T19:13:44.742Z DEBUG topgrade::runner     > Step "gcloud"
 2020-11-30T19:13:44.742Z DEBUG topgrade::runner     > Step "Flatpak"
 2020-11-30T19:13:44.742Z DEBUG topgrade::utils      > Detected "/usr/bin/flatpak" as "flatpak"

―― 22:13:44 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:44.742Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--user" "-y"
Looking for updates…
Nothing to do.
 2020-11-30T19:13:44.772Z DEBUG topgrade::executor   > Running "/usr/bin/flatpak" "update" "--system" "-y"
Looking for updates…
Nothing to do.
 2020-11-30T19:13:45.182Z DEBUG topgrade::runner     > Step "snap"
 2020-11-30T19:13:45.182Z DEBUG topgrade::runner     > Step "pihole"
 2020-11-30T19:13:45.182Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2020-11-30T19:13:45.182Z DEBUG topgrade::utils      > Detected "/usr/bin/fwupdmgr" as "fwupdmgr"
 2020-11-30T19:13:45.183Z TRACE topgrade::executor   > Output of "uname" "-r": Output { status: ExitStatus(ExitStatus(0)), stdout: "5.9.10-200.fc33.x86_64\n", stderr: "" }
 2020-11-30T19:13:45.183Z DEBUG topgrade::steps::os::linux > Uname output: 5.9.10-200.fc33.x86_64

―― 22:13:45 - Firmware upgrades ――――――――――――――――――――――――――――――――――――――――――――――――
 2020-11-30T19:13:45.183Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "refresh"
Firmware metadata last refresh: 2 hours ago. Use --force to refresh again.
 2020-11-30T19:13:45.209Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "get-updates"
Devices with no available firmware updates: 
*Redacted*
No updatable devices
 2020-11-30T19:13:45.245Z DEBUG topgrade::runner           > Step "Restarts"

―― 22:13:45 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: OK
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antigen: SKIPPED: Path "/home/mark/antigen.zsh" doesn't exist
zplug: SKIPPED: Path "/home/mark/.zplug" doesn't exist
zinit: SKIPPED: Path "/home/mark/.zinit" doesn't exist
zim: SKIPPED: Path "/home/mark/.zim" doesn't exist
oh-my-zsh: FAILED
fisher: SKIPPED: Cannot find "fish" in PATH
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/home/mark/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: SKIPPED: Cannot find "tldr" in PATH
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/home/mark/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
rustup: OK
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: OK
Flutter: SKIPPED: Cannot find "flutter" in PATH
Go: SKIPPED: Path "/home/mark/go" doesn't exist
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Cannot find "vim" in PATH
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: NPM root at /usr/local/lib/node_modules
 isn't a decandent of the user's home directory
yarn: SKIPPED: Cannot find "yarn" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Path "/home/mark/.gem" doesn't exist
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
Flatpak: OK
snap: SKIPPED: Cannot find "snap" in PATH
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: OK
Restarts: SKIPPED: Cannot find "needrestart" in PATH
 2020-11-30T19:13:45.245Z DEBUG topgrade::terminal         > Desktop notification: Topgrade finished with errors

</pre></details>

### PR DESCRIPTION
**Pull request by r-darwish**
## Standards checklist:

- [ ] The PR title is descriptive.
- [ ] The code compiles
- [ ] The code passes rustfmt
- [ ] The code passes clippy
- [ ] The code passes tests
- [ ] *Optional:* I have tested the code myself
    - [ ] I also tested that Topgrade skips the step where needed

If you developed a feature or a bug fix for someone else and you do not have the
means to test it, please tag this person here.

### CODE REVIEW

**Comment by r-darwish** [comment]
@martinxxd / @markusgod please check if this fixes the bug.

**Comment by markusgod** [comment]
This fixes problem for me. Thanks!

**Comment by martinxxd** [comment]
Working for me as well, thanks!
